__tests__/attachEventListeners.test.js

```diff
@@ -0,0 +1,89 @@
+import { JSDOM } from "jsdom";
+import { describe, expect, test, afterEach, vi, beforeAll } from "vitest";
+import { attachEventListeners, initializeElementsReferences } from "../src/script";
+import handleFormSubmit from '../src/handleFormSubmit';
+
+describe("attachEventListeners", () => {
+
+  global.chrome = {
```

Comment:
could we use [vi.stubGlobal](https://vitest.dev/api/vi.html#vi-stubglobal) here?

Reply:
Im not sure, this structure is just a placeholder for using a vi spies to inject into functions and watch them.
vi.stubGlobal looks like a global property/function sub, Im not sure how it will handle the complex chrome object that Im stubbing.
I think we can consider it a tech debt to find a better solution, but for now its the best one I could think of and it does it job.